storybook/package.json

```diff
@@ -8,10 +8,15 @@
   "private": true,
   "dependencies": {
     "@babel/core": "^7.3.4",
+    "@nuxeo/nuxeo-elements": "^3.0.0-SNAPSHOT.20190603.162100-9fc5041",
```

Comment:
Why not pulling latest?

Reply:
Dependencies on elements should be set to "3.0.0-SNAPSHOT". Please check nuxeo/nuxeo-elements#129.